source/release-notes/1.9.txt

```diff
@@ -12,16 +12,46 @@ Release Notes for mongosync 1.9
    :depth: 2
    :class: singlecol
 
-.. include:: /includes/in-dev.rst
-
 .. _1.9.0-c2c-release-notes:
 
 This page describes changes and new features introduced in  
 {+c2c-full-product-name+} 1.9 and the {+c2c-full-beta-program+}.
 
+1.9.0 Release 
+-------------
+
+**Upcoming**
+
+Other Notes 
+~~~~~~~~~~~
+
+New Features: 
+
+- ``mongosync`` now supports migrating documents that have an empty timestamp 
+  values, such as Timestamp(0,0) for source clusters using MongoDB version 6.0 
+  or later. 
+
+- Added a ``stdin`` disclaimer to the ``mongosync`` executable regarding 
+  verification. Users must to accept this disclaimer before ``mongosync``  
+  can run.
+
+- Added ``--acceptDisclaimer`` flag for users that have previously read the 
+  disclaimer to bypass it.
+
+Optimizations: 
+
+- Increased internal timeout to reduce the occurrence of timeout failures during 
+  initialization
+
+Fixed Issues: 
+
+- Fixed a bug introduced in ``mongosync`` 1.8.0 where ``mongosync`` would crash 
```

Comment:
This was already included in the 1.8.1 release notes

Reply:
Yeah I don't think we need it (even though it was included in the downstream changes)

Reply:
Gotcha. I'll remove it!